### PR TITLE
movie: The One (2001) initial review

### PR DESCRIPTION
**Pull request by paul-sheridan**
Stock martial arts actions film, but set in the multiverse. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -840,6 +840,49 @@ facing a home invader [A sadistic janitor broke into a family's home, killed the
...
+parallel universe [The story follows the former interdimensional police officer Gabriel Yulaw as he seeks to hunt down all variations of himself in alternate universes.]
+facing organized pursuit [Gabriel Yulaw was being pursued by MultiVerse Authority agents across multiple alternative realities.]
+law enforcement [Members of an agency to police the multiverse, called the MultiVerse Authority, were after one of their own who'd gone rogue and started hunting down all variations of himself in alternate universes. In one such alternate universe, the viewer is shown Gabe Law's experiences working as a police officer for the Los Angeles Police Department.]
+what if I met my double [The story followed the LAPD police officer Gabe Law as a version of him from another universe, Gabriel Yulaw, tried to hunt him down and kill him. In the end, Gabe used his martial arts skills to defeat Gabriel in an epic one-on-one fight.]
+speculative travel [A main novelty of the film is that MultiVerse Authority agents interdimensionally travel from one alternate universe to another by means of wormholes.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:872]: This is a can of wormholes. We have not been categorising wormhole traversal as speculative travel before. And if we start, surely it's a kind of space travel? Or need we include walking through magic doors and mirrors as speculative travel too?

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -840,6 +840,49 @@ facing a home invader [A sadistic janitor broke into a family's home, killed the
...
+human interpersonal relationship [The buddy police detectives Evan Funsch and Harry Rodecker were pursuing their former colleague Gabriel Yulaw across multiple alternate realities.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:873]: Time for a "close colleagues" theme?

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 3]
```diff
@@ -840,6 +840,49 @@ facing a home invader [A sadistic janitor broke into a family's home, killed the
...
+
+:: References
+https://en.wikipedia.org/wiki/The_One_(2001_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+multiverse [The story follows the former interdimensional police officer Gabriel Yulaw as he seeks to hunt down all variations of himself in alternate universes.]
+parallel universe [The story follows the former interdimensional police officer Gabriel Yulaw as he seeks to hunt down all variations of himself in alternate universes.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:868]: How does this relate to "multiverse"? Parent?

**Review by odinlake — COMMENTED** [review]
Comments inline

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:872]: A big can of wormholes, indeed. I deleted it.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 3 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:868]: Deleted the now deprecated *multiverse* theme.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:873]: Save for when we review all the unspecified *human interpersonal relationship* usages, of which there are various?

Comment (odinlake) [reply notes/stories/film/film-scifi-2000s.st.txt:871]: Ok. Is it on our list of things to do?

Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:871]: It is now!

**Pull request merged**